### PR TITLE
zip stub, nf-test, snap

### PR DESCRIPTION
**Pull request by sateeshperi**
not sure how to fix the lint fail in meta.yml re stageAs inputs

### CODE REVIEW

**Comment by jfy133** [comment]
Will need to hear about linting message from @mirpedrol and @mashehu I think... I've not seen it before either

**Comment by mashehu** [comment]
Hmm, our regex to extract the inputs does not consider stageAs yet. Please open an issue on tools.

File: modules/nf-core/zip/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.zipped_archive.collect { file(it[1]).getName() } +
+                                process.out.versions ).match() }
```
Comment (SPPearce) [inline modules/nf-core/zip/tests/main.nf.test:53]:
```suggestion
                                  process.out.versions ).match() }
```

**Review by SPPearce — APPROVED** [review]

**Comment by SPPearce** [comment]
Completed in #4663 4468

**Pull request closed without merging**